### PR TITLE
Added communications data

### PR DESCRIPTION
**Pull request by J0MS**
Added useful info about open source communication software.

### CODE REVIEW

File: projects/infrastructure/comunications/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Decentralized social network, open source.
+
+
+### [Tor network](https://www.torproject.org/)
+
+An encrypted  network using onion protocol.
+
+
+### [wireguard VPN Tunnel](https://www.wireguard.com/)
```
Comment (fabianhjr) [inline projects/infrastructure/comunications/README.md]: VPN services don't fit the scope.

File: projects/decentralized-applications/README.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ A blog publishing app for long-form articles. Also has private messaging integra
 
 # Social Networks
 
+## [Diaspora*](https://diasporafoundation.org/)
```
Comment (fabianhjr) [inline projects/decentralized-applications/README.md]: Diaspora is federated not decentralized. (Like activitypub/mastodon/etc)

**Review by fabianhjr — CHANGES_REQUESTED** [review]
Please remove the two projects out of scope or counterargue for their inclusion as decentralization projects.

File: projects/infrastructure/comunications/README.md (hunk 1 above)
Comment (J0MS) [reply projects/infrastructure/comunications/README.md]: Understood ;), I moved Wireguard into ->  projects/decentralized-applications/
```suggestion
### [wireguard VPN Tunnel](https://www.wireguard.com/)
```
It is ok, or Wireguard must be removed? What do you think?

File: projects/decentralized-applications/README.md (hunk 2 above)
Comment (J0MS) [reply projects/decentralized-applications/README.md]: Ok, removed Diaspora*

**Comment by J0MS** [comment]
@fabianhjr 
Changes made ;)

**Pull request merged**